### PR TITLE
[CFD-98] Add icons to the navbar component

### PR DESCRIPTION
**Pull request by helenaglow**
### Description

Added icons!

### Reason for Change

Completes CFD-98

### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Testing
Just run the app

### CODE REVIEW

**Review by BakerWJ — CHANGES_REQUESTED** [review]
image

Can you use the updated design for when an icon is clicked?

**Comment by helenaglow** [comment]
Ok will do when I get home 👍

**Comment by helenaglow** [comment]
https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/3f322915-8e42-476b-8cb3-c4328f153603

File: .vscode/settings.json
Comment (BakerWJ) [inline .vscode/settings.json:1]: revert

**Review by BakerWJ — CHANGES_REQUESTED** [review]
looks good -- just revert unrelated change

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**